python_modules/dagster/dagster/_core/execution/context/output.py

```diff
@@ -228,21 +212,25 @@ def run_id(self) -> str:
     @public  # type: ignore
     @property
     def metadata(self) -> Optional[Mapping[str, object]]:
+        """A dict of the metadata that is assigned to the OutputDefinition that produced the output."""
```

Comment:
`OutputDefinition` is actually a legacy class. I would say "op Out or asset definition".